recipe/meta.yaml

```diff
@@ -30,9 +35,11 @@ test:
   commands:
```

Comment:
Added, but what's this for? Does it not handle the VC features correctly otherwise?